src/Utility/HttpStatus.php

```diff
@@ -0,0 +1,150 @@
+<?php
+/**
+ * Helper class for dealing with HTTP status codes.
+ *
+ * @package Requests\Utilities
+ */
+
+namespace WpOrg\Requests\Utility;
+
+use WpOrg\Requests\Exception\InvalidArgument;
+
+/**
+ * HTTP status codes helper class.
+ *
+ * @package Requests\Utilities
+ */
```

Comment:
```suggestion
/**
 * Helper class for dealing with HTTP status codes.
 *
 * @package Requests\Utilities
 * @since   2.1.0
 */
```

For future maintenance, I think it might also be helpful to include a link to the (official/authoritive) source of the message texts.